### PR TITLE
Configure Flutter Version Management

### PR DESCRIPTION
**Pull request by Lohann**
Changes:
- [X] Use [Flutter Version Management](https://fvm.app/), so we can keep track of what version is supported in the project.
- [X] Install coverage as a local dependency
- [X] Graduate pre-release packages to release

### CODE REVIEW

**Comment by leonardocustodio** [comment]
Se voce remover o melos nao vai rodar os tests...

File: packages/polkadart/pubspec.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ dependencies:
   http: ^0.13.5 # BSD-3-Clause
   pointycastle: ^3.6.2 # MIT
   web_socket_channel: ^2.3.0 # BSD-3-Clause
-  polkadart_scale_codec: ^1.1.0-pre.2 # Apache 2.0
-  substrate_metadata: ^1.1.0-pre.1 # Apache 2.0
+  polkadart_scale_codec: ^1.1.0 # Apache 2.0
+  substrate_metadata: ^1.1.0 # Apache 2.0
```
Comment (pastre) [inline packages/polkadart/pubspec.yaml:16]: nit: fixed versions mitigate local conflicts in my experience. Can be easily done in the future :)

**Review by pastre — APPROVED** [review]
LGTM :)

**Comment by gabrielokura** [comment]
Flutter SDK uses this command to automate updates on change log. Could be interesting to this project in the future 

https://github.com/flutter/packages/tree/main/script/tool#update-changelog-and-version

**Review by justkawal — APPROVED** [review]
LGTM

**Comment by Lohann** [comment]
@gabrielokura Nice, we are using an library named [Melos](https://melos.invertase.dev/) (Thank you @leonardocustodio !), it also have a command for automatically [update the changelog of many packages at once](https://melos.invertase.dev/commands/version)!

**Review by leonardocustodio — APPROVED** [review]

File: packages/polkadart/pubspec.yaml (hunk 1 above)
Comment (Lohann) [reply packages/polkadart/pubspec.yaml:16]: And better, all those versions are automatically managed by melos :) 
- https://melos.invertase.dev/commands/version
- https://melos.invertase.dev/commands/publish

**Pull request merged**